composer.json

```diff
@@ -31,8 +31,9 @@
             "url": "https://github.com/sponsors/yiisoft"
         }
     ],
+    "minimum-stability": "dev",
```

Comment:
Remove before release.

Reply:
Most likely it will be removed in a separate PR, so I created according issue - #29.